package.json

```diff
@@ -16,7 +16,7 @@
     "deploy": "gh-pages -d docs -b master",
     "dev:all": "lerna run dev --parallel --scope \"!@blueprintjs/{landing-app,table-dev-app}\"",
     "dev:core": "lerna run dev --parallel --scope \"@blueprintjs/{core,icons,docs-app}\"",
-    "dev:docs": "lerna run dev --parallel --scope \"@blueprintjs/{docs-app,docs-theme}\"",
+    "dev:docs": "lerna run dev --parallel --scope \"@blueprintjs/{core,docs-app,docs-theme}\"",
```

Comment:
Reasoning here?

Reply:
consistency: all other `dev:*` tasks watch core. both docs-* packages depend on core and i find when i'm dev'ing docs i often want to make changes in core.